app/poros/github/results.rb

```diff
@@ -14,6 +14,12 @@ def initialize(current_user)
     @repos = parse_repos
   end
 
+  def user_info(gh_username)
```

Comment:
Not a huge fan of this.. Because the way a normal user would use our application - it would instantiate the user with the token then as opposed to us needed to instantiate another Results object within the controller, just for the sake of the test.